### PR TITLE
added data_augment.py

### PR DESCRIPTION
**Pull request by finitearth**
Second try of commiting for https://github.com/LAION-AI/Open-Assistant/issues/460

### CODE REVIEW

**Comment by finitearth** [comment]
I was able to run pre-commit now, I hope the checks run through properly now

**Review by yk — APPROVED** [review]
thank you!

File: scripts/data_augment/data_augment.py [hunk 1]
```diff
@@ -0,0 +1,464 @@
...
+from bs4 import BeautifulSoup as bs
+
+import requests
+import json
+import html
+import os
+import subprocess
+import argparse
+
+
```
Comment (andrewm4894) [inline scripts/data_augment/data_augment.py:23]: would you be able to maybe add a comment showing some example usage of the script? or some examples in a README.md in the folder if easier.

File: scripts/data_augment/data_augment.py [hunk 2]
```diff
@@ -0,0 +1,464 @@
+import torch
```
Comment (andrewm4894) [inline scripts/data_augment/data_augment.py:1]:
```suggestion
'''What does this script do

example usage: 

args: 
'''

import torch
```

maybe add some small description and example usage and maybe args at top of file so easy for others to just get quick idea of what its doing.

**Comment by finitearth** [comment]
Not sure why it keeps failing... On my command line it shows that all commits ran without fails

**Comment by yk** [comment]
> Not sure why it keeps failing... On my command line it shows that all commits ran without fails

you have to commit and push again after letting it run, as it does auto-fixes and modifies your files

File: scripts/data_augment/data_augment.py (hunk 2 above)
Comment (finitearth) [reply scripts/data_augment/data_augment.py]: Added the comment, thanks for your feedback

**Review by andrewm4894 — APPROVED** [review]

**Comment by andrewm4894** [comment]
@finitearth you should be good to merge this.

**Pull request merged**